account_banking_payment_export/views/account_payment.xml

```diff
@@ -15,11 +15,14 @@
                 </xpath>
                 <field name="mode" position="after">
                     <field name="mode_type" invisible="1"/>
+                    <field name="payment_order_type" invisible="1"/>
                 </field>
-                <xpath expr="//button[@string='Invoices']" position="attributes">
-                    <attribute name="attrs">{
-                        'invisible': [('state', '!=', 'draft')]}</attribute>
-                </xpath>
+                <button string="Invoices" position="attributes">
+                    <attribute name="attrs">{'invisible': ['|', ('state', '!=', 'draft'), ('payment_order_type', '!=', 'payment')]}</attribute>
```

Comment:
It's a verbatim copy from here: https://github.com/OCA/bank-payment/pull/169/files#diff-b5d41850b1a9531fb0e8567b9c3066b2L36

Basically, it's because there are 2 "Invoices" buttons, one for direct debits, and another for payments, and this avoids to appear the incorrect button on the non corresponding view.